### PR TITLE
[stable/goldilocks] Monitoring all namespaces out of the box

### PR DESCRIPTION
**Pull request by xogoodnow**
Updated chart version

**Why This PR?**
Most users would want to monitor all namespaces within their cluster.

**Changes**
Changes proposed in this pull request:

Adding how to monitor all namespaces right out of the box in values file of goldilocks chart

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Comment by xogoodnow** [comment]
Hi, @sudermanjr
Please let me know if there is an issue with this MR
Cheers

**Comment by xogoodnow** [comment]
Hi @sudermanjr,
Please let me know if there is something wrong with this MR
Cheers

File: stable/goldilocks/values.yaml [hunk 1]
```diff
@@ -46,8 +46,7 @@ controller:
     create: true
     # controller.serviceAccount.name -- The name of an existing service account to use for the controller. Combined with `controller.serviceAccount.create`
     name:
-
-  # controller.flags -- A map of additional flags to pass to the controller
+  # controller.flags -- A map of additional flags to pass to the controller. For monitoring all namespaces out of the box, add the follwoing flag "on-by-default: true"
```
Comment (sudermanjr) [inline stable/goldilocks/values.yaml]:
```suggestion
  # controller.flags -- A map of additional flags to pass to the controller. For monitoring all namespaces out of the box, add the following flag "on-by-default: true"
```

File: stable/goldilocks/values.yaml [hunk 2]
```diff
@@ -108,7 +107,7 @@ dashboard:
     port: 80
     # dashboard.service.annotations -- Extra annotations for the dashboard service
     annotations: {}
-  # dashboard.flags -- A map of additional flags to pass to the dashboard
+  # dashboard.flags -- A map of additional flags to pass to the dashboard. For monitoring all namespaces out of the box, add the follwoing flag "on-by-default: true".
```
Comment (sudermanjr) [inline stable/goldilocks/values.yaml]:
```suggestion
  # dashboard.flags -- A map of additional flags to pass to the dashboard. For monitoring all namespaces out of the box, add the following flag "on-by-default: true".
```

**Review by sudermanjr — APPROVED** [review]
Thanks!

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Please update the readme per the contributing guide - https://github.com/FairwindsOps/charts/blob/master/CONTRIBUTING.md#documentation

**Comment by xogoodnow** [comment]
Hi @sudermanjr, 
Readme file has been updated.
If there is anything else please let me know.
Cheers

**Review by sudermanjr — APPROVED** [review]

**Pull request merged**